### PR TITLE
79 Fix redirection when user isn't connected with facebook/google button

### LINKED ISSUE #79: [BUG]: Facebook button automatically redirect to home page without connection
### Project location

Application

### What happened?

The facebook button in the sign up page redirect to home page without any connection.

### How to reproduce?

- Go to sign up page
- Click on facebook button

### Relevant log output

_No response_

### PR DESCRIPTION
**Pull request by CedricCORGE**
# Bug fixed
Remove the possibility to dodge authentification processus by using google/facebook signin button

Closes #79

### CODE REVIEW

**Review by STom6 — DISMISSED** [review]
lgtm

**Review by JohanCDev — COMMENTED** [review]
lgtm

**Review by JohanCDev — DISMISSED** [review]

File: Application/src/Screens/ConnexionScreen.tsx [hunk 1]
```diff
@@ -115,11 +115,11 @@ export default function ConnexionScreen() {
 
     function connectWithGoogle() {
         console.log("Connect with Google")
-        const props: HomeScreenProps = {
+        /*const props: HomeScreenProps = {
             userMail: userMail,
             userId: "idTest"
         }
-        NavigatorPush("HomeScreen", "mainStack", options, props)
+        NavigatorPush("HomeScreen", "mainStack", options, props)*/
```
Comment (osvegn) [inline Application/src/Screens/ConnexionScreen.tsx]: Why is it kept as comment ?

File: Application/src/Screens/SignInScreen.tsx [hunk 2]
```diff
@@ -68,20 +72,40 @@ export default function SignInScreen() {
 
     function connectWithGoogle() {
         console.log("Subscribe with Google")
-        const props: HomeScreenProps = {
+        /*const props: HomeScreenProps = {
             userMail: userMail,
             userId: "idTest"
         }
-        NavigatorPush("HomeScreen", "mainStack", options, props)
+        NavigatorPush("HomeScreen", "mainStack", options, props)*/
```
Comment (osvegn) [inline Application/src/Screens/SignInScreen.tsx]: Why is it kept as comment ?

File: Application/src/Screens/SignInScreen.tsx [hunk 3]
```diff
@@ -68,20 +72,40 @@ export default function SignInScreen() {
...
         }
-        NavigatorPush("HomeScreen", "mainStack", options, props)
+        const data = await AccessToken.getCurrentAccessToken();
+
+        if (!data) {
+            throw 'Something went wrong obtaining access token';
+        }
+
+        // Create a Firebase credential with the AccessToken
+        console.log(data.accessToken)
```
Comment (osvegn) [inline Application/src/Screens/SignInScreen.tsx]: Useful to log an access token ?

**Review by osvegn — CHANGES_REQUESTED** [review]
Some questions about code and comments

**Review by 31Nathan — APPROVED** [review]
LGTM

**Review by osvegn — APPROVED** [review]
Look good

**Pull request merged**